src/libraries/System.Net.Quic/src/Resources/Strings.resx

```diff
@@ -150,5 +150,8 @@
   <data name="net_quic_writing_notallowed" xml:space="preserve">
     <value>Writing is not allowed on stream.</value>
   </data>
+  <data name="net_ssl_app_protocols_invalid" xml:space="preserve">
```

Comment:
do we need this? looks like extra.

Reply:
Yes we need this, it's used by one of the included interop source files:
https://github.com/dotnet/runtime/blob/d148c34bdfbd49e3d820eaa4dcbc47832d52d0d2/src/libraries/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.SslCtx.cs#L43-L43

https://github.com/dotnet/runtime/pull/56175/files#diff-1d4ce147edca0ce81f312b96fce76ac52eb02eb6d04e660817e3b47230edc794R64